src/oncall/api/v0/ical_key_team.py

```diff
@@ -50,8 +50,12 @@ def on_post(req, resp, team):
     challenger = req.context['user']
     check_calendar_auth(team, req)
 
-    update_ical_key(challenger, team, 'team', str(uuid.uuid4()))
```

Comment:
Can you abstract out the fact that we're using `str(uuid.uuid4())` for our ical key out into a utility function and use that any time we need to generate a new key?

Something like:
```
def generate_ical_key():
    return str(uuid.uuid4())
```

That way in the event that we decide to change the format of newly generated keys from UUIDs to something different, we would only need to change it in one place!

Reply:
Good Point!